u3-views/app.js

```diff
@@ -0,0 +1,34 @@
+const express = require('express')
+const expHbs = require('express-handlebars')
+const {index} = require('./features/indexController')
+const { courseDetails,courseUnits,courseStudents } = require('./features/courseController')
+const { videoDetails1,videoDetails2 } = require('./features/videoController')
+
+const app = express()
+
+// Templates
+app.set('views', './views')
+app.set('view engine', 'hbs')
+app.engine('hbs', expHbs({
+  extname: '.hbs',
+  defaultLayout: false,
+  partialsDir: ['./views/partials', './views/layouts']
+}))
+
+
+// Middleware
+app.use(express.static('public'))
+
+// Routes
+app.get('/', index)
+app.get('/courses/:code', courseDetails)
+app.get('/courses/:code/units', courseUnits)
+app.get('/courses/:code/students', courseStudents)
+app.get('/videos/128', videoDetails1)
```

Comment:
`videoId` จะไม่เป็น `128` เสมอไป คุณต้องใช้ "พารามิเตอร์เส้นทาง" (ตัวแปรภายในเส้นทางของคุณ) รหัสที่ถูกต้องคือ:
The `videoId` won't always be `128`. You need to use a "path parameter" (a variable within your route). The correct code is:
```js
app.get('/videos/:videoId', ...)
```